docs/tilt.md

```diff
@@ -48,6 +48,10 @@ kustomize_substitutions:
   HC_NODE_MACHINE_TYPE: "x1e.2u.4g"
 
   ECS_IMAGE_ID: "a9f5cc27-0d50-4864-a45e-d8eead734a3f"
+
+  KUBE_IMAGE_REPO: "registry.cn-hangzhou.aliyuncs.com/google_containers"
+  KUBE_DNS_IMAGE_REPO: "registry.cn-hangzhou.aliyuncs.com/google_containers"
+  ETCD_IMAGE_REPO: "registry.cn-hangzhou.aliyuncs.com/google_containers"
```

Comment:
Does aliyun sync latest patches and new versions? 
Or we may use `m.daocloud.io`.